src/ui/components/shared/SharingModal/CollaboratorsList.css

```diff
@@ -6,6 +6,19 @@
   max-height: 240px;
 }
 
+.permissions-list {
+  animation: fadeIn 0.4s ease-in-out;
+}
```

Comment:
I think my earlier comment is still relevant here:
https://github.com/replayio/devtools/pull/8541/files/5273ffda3a10103876478a8a58eee869a61f9fd5#diff-380e2a93729a0b270e3bc9f8aeb3af1a097d967993100c0326b42f70eb794334

This style was _added_ in this PR and it's definitely _not_ being referenced, so we should remove it (and preferably the one above it)

Reply:
Feel free to squash and merge if I'm asleep and we get on the same page with the animation.

Reply:
Okay. I should have checked out your branch and searched it locally before commenting.

You're right. There is one component still referencing the _global_ class name:
```jsx
export default function CollaboratorsList({
  owner,
  collaborators,
}: {
  owner: User;
  collaborators: CollaboratorDbData[] | null;
}) {
  return (
    <div className="permissions-list overflow-auto">
      <Author user={owner} />
      {collaborators ? <Collaborators {...{ collaborators }} /> : null}
    </div>
  );
}
```

My [initial comment](https://github.com/replayio/devtools/pull/8541/files/5273ffda3a10103876478a8a58eee869a61f9fd5#diff-380e2a93729a0b270e3bc9f8aeb3af1a097d967993100c0326b42f70eb794334) was on a different file and I think I 
mixed the two up when I saw the `permissions-list` style. My apologies.